visualization/app/codeCharta/ui/codeMap/threeViewer/threeRendererService.ts

```diff
@@ -19,17 +24,93 @@ export class ThreeRendererService implements IsWhiteBackgroundSubscriber {
 	static RENDER_OPTIONS = {
```

Comment:
If you do it like this,
```suggestion
	static RENDER_OPTIONS: WebGLContextAttributes = {
```
not specified attributes will not be set to a valid default value as "as WebGLContextAttributes" will do (i'ts like casting).